src/styles/blocks/card.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-top: 32px;
+    margin-left: 20px;
+    margin-bottom: 0;
+    display: block;
+  }
+
+  &__title {
+    margin: 40px 16px 0;
```

Comment:
You don`t need to set in each element 16px inline-margins, better set in block card paddings 16px